src/h5web/vis-packs/core/utils.ts

```diff
@@ -218,11 +218,15 @@ export function getCanvasScale(
   config: AxisConfig,
   canvasSize: number
 ): AxisScale {
-  const { scaleType, visDomain } = config;
+  const { scaleType, visDomain, flip } = config;
+  const range = [-canvasSize / 2, canvasSize / 2];
+  if (flip) {
+    range.reverse();
+  }
 
   return createAxisScale({
     domain: visDomain,
-    range: [-canvasSize / 2, canvasSize / 2],
+    range,
```

Comment:
This inverts the scale. As a consequence, all components that rely on the scale will follow the inversion, namely:
- `Axes`
- `TooltipMesh`
- `DataCurve`
- `Annotation`

Reply:
`range: flip ? range.reverse() : range`